components/tools/tools-display.tsx

```
@@ -0,0 +1,177 @@
+                className="grid grid-cols-1 md:grid-cols-2 gap-3 mt-3"
+              >
+                {categoryTools.map(tool => (
+                  <button 
+                    key={tool.name}
+                    onClick={() => handleToolClick(tool.name)}
+                    className={cx(
+                      'p-4 rounded-lg border shadow-sm hover:shadow-md transition-shadow text-left',
+                      bgColor.replace('100', '50'),
+                      'hover:bg-opacity-80'
+                    )}
+                  >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)